service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/DocmosisTemplateImages.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.fpl.model.docmosis;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class DocmosisTemplateImages {
```

Comment:
Thoughts on expanding this slightly, instead of having a builder we "delombok" that part and we then change the signatures so that they can be something like
`.withDraftWaterMark()` which then points to the draft watermark constant (which could then be removed from the enum and made a constant property in this class)